earthpy/io.py

```diff
@@ -117,51 +136,55 @@ def get_data(self, key=None, replace=False, url=None):
         -------
         path_data : str
             The path to the downloaded data.
+
+        Examples
+        --------
+        Download a dataset using a key:
+
+            >>> earthpy.data.get_data('california-rim-fire') # doctest: +SKIP
```

Comment:
Let's be consistent with calling this
```
import earthpy as et
et.data.get_data('california-rim-fire')
```

Reply:
Sure - I changed the examples to use `et.data`.